agent/src/capability/capability.cpp

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Imho this capability class does not make much sense with the current setup. HOWEVER, if we want to use it to merge gateway configurations together, it would start to make more sense though. And it is nice to capture things in classes anyway.